lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSignupCard.js

```diff
@@ -66,26 +66,29 @@ class CampaignSignupCard extends React.Component {
   }
 
   render() {
+    const header = this.renderHeader();
     return (
-      <article className="beta-card">
-        <div className="beta-card__body">
+      <li>
+        <article className={`beta-card ${header === null ? '-headerless' : ''}`}>
+          <div className="beta-card__body">
 
-          {this.renderHeader()}
+            {header}
```

Comment:
I think it was fine before w/out saving it to a `const` and just calling `{this.renderHeader()}`. You don't seem to be using it elsewhere, where it would warrant reuse?